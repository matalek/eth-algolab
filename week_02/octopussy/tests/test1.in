17
7
7 6 5 1 2 8 9
3
3 1 1
7
7 10 10 8 8 6 7
7
7 3 3 19 7 9 1
7
7 6 15 1 11 12 2
7
7 5 3 7 10 9 1
7
7 3 8 12 9 10 1
7
7 3 4 3 1 1 1
7
7 3 9 6 2 9 6
7
7 13 6 2 10 12 15
7
7 5 5 5 10 12 1
7
7 4 3 9 9 7 1
7
13 3 3 18 3 9 15
7
7 3 12 1 7 1 10
7
7 3 11 4 10 8 12
7
7 16 7 12 8 5 1
7
9 4 19 5 9 15 14
