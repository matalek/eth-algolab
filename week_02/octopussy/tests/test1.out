yes
no
yes
no
yes
no
no
no
yes
yes
no
no
no
no
yes
yes
yes
